spec/cli.spec.js

```diff
@@ -497,4 +497,16 @@ describe('cordova cli', () => {
         });
 
     });
+
+    describe('requirements', () => {
+        beforeEach(() => {
+            spyOn(cordova, 'requirements').and.returnValue(Promise.resolve({browser: []}));
+        });
+
+        it('Test#050 : requirements browser', () => {
```

Comment:
I feel this format would be better and preferred. I know you are following the existing format of the test structure so it is your decision if you want to accept this.

As for everything else, it is OK and can be merged into `master`.

```suggestion
        it('should pass browser as the platform argument to the requirement check method.', () => {
```

This makes it more human readable when the test fails and tells the user what `it` is expected.